packages/ripple-nuxt-tide/modules/webform/components/TideContentRating.vue

```diff
@@ -23,6 +23,9 @@ export default {
   components: {
     RplForm
   },
+  props: {
+    siteSectionName: String
```

Comment:
Can we add a default value for it?
```
siteSectionName: {
  type: String,
  default: ''
}
```

https://vuejs.org/v2/guide/components-props.html#Prop-Validation